src/handlers/issue/comment-scoring-rubric.ts

```diff
@@ -159,10 +159,17 @@ export class CommentScoring {
   ): (typeof CommentScoring.prototype.commentScores)[number]["details"][number]["wordScoreCommentDetails"] {
     const wordScoreCommentDetails: { [key: string]: Decimal } = {};
 
+    const builtIns = new Set(
+      Object.getOwnPropertyNames(Object.prototype).filter((name) => typeof Object.prototype[name] === "function")
+    );
+
     for (const word of words) {
-      let counter = wordScoreCommentDetails[word] || ZERO;
-      counter = counter.plus(this.roleWordScore);
-      wordScoreCommentDetails[word] = counter;
+      if (!builtIns.has(word)) {
+        let counter = wordScoreCommentDetails[word] || ZERO;
```

Comment:
`ZERO` it's still constructed as a Decimal, can you perhaps so it's expect a decimal in the end, can you perhaps type `let counter: Decimal` as test as such?

Reply:
Good observation

Reply:
```ts
const counter: Decimal
```

It's already inferred as such without being explicit but the test still passes either way, I have just tried locally but with the PR being closed it's a dud.

Proved to be working via rpc-handler anyway happy days

Reply:
perhaps for a future improved, pavlovick is one of the like loves everything `Typed`!